Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/PsychEyelinkDispatchCallback.m

```diff
@@ -309,7 +309,11 @@
 % Draw calibration target, if any is specified:
 if ~isempty(calxy)
     drawInstructions=0;
-    EyelinkDrawCalibrationTarget(eyewin, el, calxy);
+    if isa(el.calTargetDrawer,'function_handle')
```

Comment:
Please use consistent spacing and formatting. A space is missing between el.calTargetDrawer, and 'function_handle'. Ideally we'd fixup more inconsistencies across the codebase, but that would clutter the git commit's with white-space changes, and getting the code style consistent is probably a battle we long lost. And i have my slip-ups in this area as well, sometimes i cave to the time pressure. Tradeoff's...

But for new code we should be at least consistent with the style that is there, or fixup the old code if it is too horrendous to bear. Otherwise it reduces readability further and makes it look "cheap".